robotoff/workers/tasks/import_image.py

```diff
@@ -215,6 +227,68 @@ def run_nutrition_table_object_detection(product_id: ProductIdentifier, image_ur
 }
 
 
+def run_upc_detection(product_id: ProductIdentifier, image_url: str):
+    """Detect the presence of a UPC in an image and determine if it dominates the image (i.e. if it is the primary content in the image)
+    :param product_id: identifier of the product
+    :param image_url: URL of the image to use
+    """
```

Comment:
Can you make sure your docstring length is < 80 characters?

Reply:
Done!